docs/stress-test.md

```diff
@@ -3,6 +3,14 @@
 This document gives an overview of the [hub-stress-test script](../scripts/hub-stress-test.py)
 and how it can be used.
 
+1. [Setup](#setup)
+1. [Scaling up](#scaling-up)
+   1. [Placeholders and user nodes](#placeholders)
+   1. [Steady state testing](#steady-state)
+1. [Scaling down](#scaling-down)
```

Comment:
I screwed this up in the merge to resolve the conflict on `Activity update testing`, will fix.